src/bin/pytom_create_template.py

```diff
@@ -68,7 +68,7 @@ def main():
     input_data = read_mrc(args.input_map)
     input_meta_data = read_mrc_meta_data(args.input_map)
     if args.input_voxel_size_angstrom is not None:
-        if args.input_voxel_size_angstrom != input_meta_data['voxel_size']:
+        if args.input_voxel_size_angstrom != round(input_meta_data['voxel_size'], 3):
```

Comment:
for sanity I would round both sides:
```suggestion
        if round(args.input_voxel_size_angstrom, 3) != round(input_meta_data['voxel_size'], 3):
```